omegaconf/_utils.py

```diff
@@ -67,7 +69,11 @@ def str_representer(dumper: yaml.Dumper, data: str) -> yaml.ScalarNode:
         )
 
 
-OmegaConfDumper.add_representer(str, OmegaConfDumper.str_representer)
+def get_omega_conf_dumper() -> Type[OmegaConfDumper]:
+    if not OmegaConfDumper.str_representer_added:
+        OmegaConfDumper.add_representer(str, OmegaConfDumper.str_representer)
+        OmegaConfDumper.str_representer_added = True
+    return OmegaConfDumper
```

Comment:
You can create a sort of static initializer as described here:
https://softwareengineering.stackexchange.com/questions/380210/why-isnt-there-a-static-initializer-in-python

Reply:
add_representer is a classmethod so it doesn't work, I've tried :/

Reply:
Sigh. I just commented here: https://github.com/yaml/pyyaml/issues/221#issuecomment-658485897

For now do the best you can.